csharp/autobuilder/Semmle.Autobuild.CSharp/StandaloneBuildRule.cs

```diff
@@ -11,10 +11,15 @@ public BuildScript Analyse(Autobuilder builder, bool auto)
         {
             BuildScript GetCommand(string? solution)
             {
-                if (builder.SemmlePlatformTools is null)
+                string standalone;
+                if (!(builder.CodeQLExtractorLangRoot is null) && !(builder.CodeQlPlatform is null)) {
```

Comment:
We have started writing `X is object` instead of `!(X is null)` (C# 9 will eventually support `X is not null`).

Reply:
Ok, changed! That looks a little nicer indeed.